Positions/SeniorSoftwareEngineer.md

```diff
@@ -1,36 +1,102 @@
-## What does "Senior Software Engineer" mean?
+## Senior Software Engineer (SSE)
 
-N.b. Not every single one of these criteria needs to be satisfied to be considered senior. But seniority comprises, in some combination or other, the majority of these.
+_Not every single one of these criteria needs to be satisfied to be considered a full JSE. But the role comprises, in some combination or other, the majority of these._
```

Comment:
"a full JSE" -> "a full SSE"

Reply:
Ah, d'oh. I actually want to keep the original line. Just without the "N.b"